src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -84,6 +84,36 @@ public static class ConfigurationKeys
         /// <seealso cref="TracerSettings.LogsInjectionEnabled"/>
         public const string LogsInjectionEnabled = "DD_LOGS_INJECTION";
 
+        /// <summary>
+        /// Configuration key for setting the number of traces allowed
+        /// to be submitted per second.
+        /// </summary>
+        /// <seealso cref="TracerSettings.MaxTracesSubmittedPerSecond"/>
+        public const string MaxTracesSubmittedPerSecond = "DD_MAX_TRACES_PER_SECOND";
+
+        /// <summary>
+        /// Configuration key for setting custom sampling rules based on regular expressions.
+        /// Semi-colon separated list of sampling rules.
+        /// The rule is matched in order of specification. The first match in a list is used.
+        ///
+        /// Per entry:
+        ///   The item 'rate' is required in decimal format.
+        ///   The item 'service' is optional in regular expression format, to match on service name.
+        ///   The item 'operation' is optional in regular expression format, to match on operation name.
+        ///
+        /// Example:
+        ///   'rate=0.5, service=cart.* ; rate=0.2, operation=web ; rate=1.0, service=background, operation=insert ; rate=0.1'
```

Comment:
is `operation=web` a real example? same with `insert`?

might be better to use `sql.request`, `http.request`, `aspmvc.request`, etc etc

Reply:
it might be worth to break these into smaller examples? seems like a long line to parse/understand.

e.g.

```
'rate=0.5'

- Use a sample rate of 50% on all root spans

'rate=0.1, service=cart.* ; rate=0.5'

- Use sample rate of 10% on any spans with a service matching the regex cart.*
- Use sample rate of 50% of all other spans

... etc etc
```

Reply:
Sure, that would be good to break up.

Reply:
> might be better to use `sql.request`, `http.request`, `aspmvc.request`, etc

I agree that these are better examples, thanks. This does raise another question. Since these are regular expressions, the more correct versions would be `sql\.request` (escaping the dot which otherwise matches anything).

I'm starting to wonder if we should let users decide if they want to use regexes or not. Maybe separate setting names like `operation` and `operationRegex`?
- `operation=sql.query` (exact match) vs `operationRegex=^sql\..+` (regex match)


Or maybe keep it simple and only allow wildcards (`*`, `?`) instead of full regex:
- `operation=sql.query` (exact match) vs `operation=sql.*` (wildcard match)

Reply:
the `.` will still match a `.`
I think this is a docs issue with low risk.

Reply:
> the `.` will still match a `.`

True, but it will also match other characters. Any change that regex is overkill here? Changing it later would be a breaking change.